src/app/app.component.html

```diff
@@ -1 +1,2 @@
-<app-header/>
\ No newline at end of file
+<app-header/>
+<app-categories></app-categories>
```

Comment:
Choose one style for component  
I think better 
<app-header></header>
<app-categories></app-categories>

Reply:
+